src/Spd.Resource.Repository/Contact/Mappings.cs

```diff
@@ -43,19 +43,19 @@ public Mappings()
             .ForMember(d => d.spd_middlename1, opt => opt.MapFrom(s => StringHelper.ToTitleCase(s.MiddleName1)))
             .ForMember(d => d.spd_middlename2, opt => opt.MapFrom(s => StringHelper.ToTitleCase(s.MiddleName2)))
             .ForMember(d => d.telephone1, opt => opt.MapFrom(s => s.PhoneNumber))
-            .ForMember(d => d.address1_line1, opt => opt.MapFrom(s => s.MailingAddress == null ? null : s.MailingAddress.AddressLine1))
-            .ForMember(d => d.address1_line2, opt => opt.MapFrom(s => s.MailingAddress == null ? null : s.MailingAddress.AddressLine2))
-            .ForMember(d => d.address1_city, opt => opt.MapFrom(s => s.MailingAddress == null ? null : s.MailingAddress.City))
-            .ForMember(d => d.address1_country, opt => opt.MapFrom(s => s.MailingAddress == null ? null : s.MailingAddress.Country))
-            .ForMember(d => d.address1_stateorprovince, opt => opt.MapFrom(s => s.MailingAddress == null ? null : s.MailingAddress.Province))
-            .ForMember(d => d.address1_postalcode, opt => opt.MapFrom(s => s.MailingAddress == null ? null : s.MailingAddress.PostalCode))
+            .ForMember(d => d.address1_line1, opt => opt.MapFrom(s => s.MailingAddress == null ? string.Empty : s.MailingAddress.AddressLine1))
+            .ForMember(d => d.address1_line2, opt => opt.MapFrom(s => s.MailingAddress == null ? string.Empty : s.MailingAddress.AddressLine2))

```

Comment:
I just check it more seriously, it is not correct. the mailingAddress could not be null, but addressline2 could be null. Then the bug is still not fixed.